src/core/bios.cpp

```diff
@@ -47,9 +47,14 @@ static constexpr std::array<ImageInfo, 21> s_image_infos = {{
    MakeHashFromString("417b34706319da7cf001e76e40136c23")},
   {"SCPH-1002, DTL-H1202, H3002 (v2.2 12-04-95 E)", ConsoleRegion::PAL,
    MakeHashFromString("e2110b8a2b97a8e0b857a45d32f7e187")},
+  {"SCPH-1002, DTL-H1100 (Version 2.2 03/06/96 D)", ConsoleRegion::PAL,
+   MakeHashFromString("ca5cfc321f916756e3f0effbfaeba13b")},
   {"SCPH-3000, DTL-H1000H (v1.1 01-22-95)", ConsoleRegion::NTSC_J,
    MakeHashFromString("849515939161e62f6b866f6853006780")},
+  {"SCPH-3000, DTL-H1001 (v2.0 05/07/95 A)", ConsoleRegion::NTSC_U,
+   MakeHashFromString("dc2b9bf8da62ec93e868cfd29f0d067d")},
```

Comment:
Based on the naming scheme, this is an NTSC-U BIOS, so I'm not sure why it would be marked as coming from an NTSC-J retail model.